src/libs/ajax/WorkspaceDataService.test.ts

```diff
@@ -0,0 +1,123 @@
+import { asMockedFn } from '@terra-ui-packages/test-utils';
+import { Ajax } from 'src/libs/ajax';
+
+import { fetchWDS } from './ajax-common';
+import { Capabilities } from './WorkspaceDataService';
+
+type AjaxCommonExports = typeof import('src/libs/ajax/ajax-common');
+
+jest.mock('src/libs/ajax/ajax-common', (): AjaxCommonExports => {
+  return {
+    ...jest.requireActual('src/libs/ajax/ajax-common'),
+    fetchWDS: jest.fn(),
+  };
+});
+
+describe('WorkspaceData', () => {
+  describe('getCapabilities', () => {
+    type SetupOptions = {
+      stubbedCapabilitiesJson?: string;
+      stubbedCapabilitiesResponse?: Response;
+      wdsProxyUrl?: string;
+    };
+    type SetupResult = {
+      wdsProxyUrl: string;
+    };
+    function setup({
```

Comment:
This setup was a big struggle to get working (particularly to mock the HTTP call correctly) and in the end feels very heavyweight.  Open to suggestions to slim it down or make it more idiomatic.